src/components/VipCore/vipCore.client.js

```diff
@@ -73,7 +74,40 @@ export async function libraryListFromPosition(
   }
 
   if (response.statusCode === 200) {
-    return parseFindLibraryResponse(response.body);
+    const checkLibraries = await getBorrowercheckLibraries();
+    return parseFindLibraryResponse(response.body, checkLibraries);
+  }
+
+  throw new VipCoreError(response.statusMessage);
+}
+
+/** return an array with agencies that support borrowerCheck from login.bib.dk
+ *
+ * @returns {Array}
+ */
+async function getBorrowercheckLibraries() {
+  if (CONFIG.mock_externals.vipCore) {
+    return [];
+  }
+    let response;
+    response = await promiseRequest('post', {
+      url: CONFIG.vipCore.uri + '/borrowerchecklist/',
+      body: {
+        serviceRequester: 'login.bib.dk',
+        borrowerCheckAllowed: 'true',
+        trackingId: 'login.bib.dk'
+      },
+      json: true
+    });
+
+  if (response.statusCode === 200) {
+    if (response.body && Array.isArray(response.body.borrowerCheckLibrary)) {
+      const checkLibraries = [];
+      response.body.borrowerCheckLibrary.forEach(agency => {
+        checkLibraries.push(agency.isil.toLowerCase().replace('dk-', ''));
```

Comment:
Kan man være sikker på at alle agencies altid har et isil nummer på sig? ellers kan den nok godt fejle fælt, men ikke sikkert at det er et problem, bare en tanke.

Reply:
Det har de pt, men hvad fremtiden kan bringe vides ikke. Retter så det tager højde for tomt/manglende isil.
Godt spottet